index.js

```diff
@@ -30,8 +31,12 @@ exports.parse = function parseIntelHex(data, bufferSize) {
 		highAddress = 0, //upper address
 		startSegmentAddress = null,
 		startLinearAddress = null,
+		addressOffset = 0 || addressOffset,
 		lineNum = 0, //Line number in the Intel Hex string
 		pos = 0; //Current position in the Intel Hex string
+
+	buf.fill(EMPTY_VALUE);
```

Comment:
Why is this needed?  Can it be avoided?

Reply:
Originally I didn't include it and the buffer's end was populated with 0s instead of FFs, so I pre-filled it with FFs to resolve, but maybe there is a better place to implement it